apps/src/lab2/views/components/versionHistory/VersionHistoryDropdown.tsx

```diff
@@ -95,13 +93,6 @@ const VersionHistoryDropdown: React.FunctionComponent<
     }
   }, [versionList, selectedVersion, latestVersion]);
 
-  const resetVersionState = useCallback(() => {
```

Comment:
I moved this logic to `ProjectContainer`, which was a more logical place for it now that we also have a `hasEdited` state to reset.